### ISSUE TITLE
NameError: global name 'arg' is not defined

### ISSUE THREAD

**Issue by epfremmer** [josegonzalez/python-github-backup#69]
Ran into a python error while testing backing up my organization's repositories.

```
# github-backup $ORGANIZATION --output-directory ./ --organization --token $GITHUB_ACCESS_TOKEN --all --private --fork --prefer-ssh --name-regex $REPO_NAME
Backing up user <REDACTED> to /root/github_backups
Retrieving repositories
Filtering repositories
Backing up repositories
Traceback (most recent call last):
  File "/usr/local/bin/github-backup", line 885, in <module>
    main()
  File "/usr/local/bin/github-backup", line 880, in main
    backup_repositories(args, output_directory, repositories)
  File "/usr/local/bin/github-backup", line 555, in backup_repositories
    lfs_clone=arg.lfs_clone)
NameError: global name 'arg' is not defined
```

It looks like there is a reference to an undefined variable in:
 - https://github.com/josegonzalez/python-github-backup/blob/0.14.0/bin/github-backup#L555
 - https://github.com/josegonzalez/python-github-backup/blob/0.14.0/bin/github-backup#L564

Probably just a typo `arg` instead of `args`. I tested changing this to args and the backup script worked great!

**Comment by josegonzalez** [comment]
Mind filing a pr with the fix? Thanks.

/cc @pieterclaerhout

**Comment by epfremmer** [comment]
no problem

**Comment by josegonzalez** [comment]
Closed by #70
